hashmap.py

```diff
@@ -0,0 +1,105 @@
+# Time Complexity : 
+# Space Complexity : 
+# Did this code successfully run on Leetcode :
+# Any problem you faced while coding this :
+
```

Comment:
use the above template and mention the time and space complexities to all the problems.